### ISSUE TITLE
After Password chage it is not working

### ISSUE THREAD

**Issue by chittaranjan870** [texxasrulez/old_calendar#25]
Dear,
I have integrated this plugin in roundcube  1.4.9 but after user password change it is not working. Require password update manual  user password in edit-caldav setings.

**Comment by texxasrulez** [comment]
The passwords must match. Was the password change done on both email and caldav server?

**Comment by chittaranjan870** [comment]
Yes password is same in both caldav and email because both uses one ldap authentication.
Direct caldav access is also working with same password. But your calendar plugin requires again  password update in caldav setting.

**Comment by texxasrulez** [comment]
I guess I am not getting what you are trying to say by  "edit-caldav setings". Do you have error messages? Screenshots of what is going on?

**Comment by chittaranjan870** [comment]
![Screenshot from 2023-04-29 18-47-15](https://user-images.githubusercontent.com/129391527/235304845-bfdb7bcf-daaa-4487-a679-e16b15e739fb.png)

Hello !!!
Hope you are doing well

I guess I couldn't explain properly. Actually, whenever I am resetting my password the calendar is not working properly. But if I come to Digest Login and give the new password and save it, it's working fine.
Currently I want the calendar should work without having to change it again in digest login section(Attached the screenshot). Please suggest.
